modules/oe_content_publication/config/post_updates/00003_create_fields/field.field.node.oe_publication.body.yml

```diff
@@ -0,0 +1,24 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.body
+    - node.type.oe_publication
+  module:
+    - text
+_core:
+  default_config_hash: epQrM0F0rqe-Y_2pNLD5jzqSV_ZAWtwN2rfls_xQIno
+id: node.oe_publication.body
+field_name: body
+entity_type: node
+bundle: oe_publication
+label: 'Body text'
+description: ''
+required: false
+translatable: false
+default_value: {  }
+default_value_callback: ''
+settings:
+  display_summary: 0
+  required_summary: false
+field_type: text_long
```

Comment:
```suggestion
settings:
  display_summary: false
  required_summary: false
field_type: text_with_summary
```

Reply:
@22Alexandra the task describes the Body field as `Type: long text (without summary)` . Why should we modify the field_type?

Reply:
This is chnaged.